fili-core/src/main/java/com/yahoo/bard/webservice/data/PartialDataHandler.java

```diff
@@ -3,14 +3,14 @@
 package com.yahoo.bard.webservice.data;
 
 import static com.yahoo.bard.webservice.config.BardFeatureFlag.PERMISSIVE_COLUMN_AVAILABILITY;
-import static com.yahoo.bard.webservice.util.TableUtils.getColumnNames;
```

Comment:
I don't see why we want to statically import this method here. Let me know if there is a reason for it. (the reason I removed it is because the IDE does not know exactly which version of `getColumnNames` the static import refers to and thinks that all versions of this function are used in this file)